_usage/hint_mapping.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Using Hint Mapping
+nav_order: 9
+---
+
+## Using Query Hint Mapping
+
+Support for query hint mapping is available with Babelfish version 2.3.0. 
+
+Query optimizers do their best to select the best execution plan for your workload based on an engine’s cost model and column/table statistics, but real-world datasets show that query optimizers sometimes don't make the best choices. You can use a query hint to encourage the query optimizer to avoid performance issues caused by bad execution plans.  
+
+A query hint instructs the database engine how to execute a query. Babelfish can use the [pg_hint_plan](https://github.com/ossc-db/pg_hint_plan) extension to create and manage query hints.
+
+
+### Installing pg_hint_plan
+
+Babelfish version 2.3.0 supports version 1.4 of [pg_hint_plan](https://github.com/ossc-db/pg_hint_plan/releases/tag/REL14_1_4_0). To install pg_hint_plan:
+
+1. Download the  `pg_hint_plan14` module. 
+2. Navigate to the top of the source tree. 
+3. Assume the identity of the cluster owner, and run `make`, and then `make install`. 
```

Comment:
This is not correct. According to https://github.com/ossc-db/pg_hint_plan#installation
```
Simply run make at the top of the source tree, then make install as an appropriate user.
```

Reply:
Updated to read:


1. Assume the identity of a sufficiently privileged user.
2. At the top of the source tree, run `make` and then `make install`. 
3. Modify your PATH environment variable to include the Babelfish PostgreSQL installation.

Reply:
There really isn't such a thing as an "appropriate" user, so I used "sufficiently privileged" user - if you have a better suggestion, I'm open to ideas...